fix: remove 32-bit binaries and ipfs-updater (#1996)
ipfs/ipfs-docs: docs/install/command-line.md

- If an `$IPFS_PATH` isn't in the default location, use the `--api <rpc-api-addr>` command-line argument. Alternatively, you can set the environment variable to `IPFS_PATH`. `IPFS_PATH` will point to a directory with the `$IPFS_PATH/api` file pointing at the Kubo RPC of the existing `ipfs` daemon instance.

::: tip

> If you plan to expose safe subset of RPC API to the public internet with TLS encryption and HTTP authentication, check out the [TLS and HTTP Auth for Kubo with Caddy](../how-to/kubo-rpc-tls-auth.md) guide.

Reviewer: ```suggestion
If you plan to expose the RPC API to the public internet with TLS encryption and HTTP authentication, check out the [TLS and HTTP Auth for Kubo with Caddy](../how-to/kubo-rpc-tls-auth.md) guide.
```

> If you are looking for HTTP Fetch API designed for browsers and public internet, with [proper HTTP Cache Control](https://specs.ipfs.tech/http-gateways/path-gateway/#cache-control-response-header), see implementation-agnostic [HTTP Gateway](../reference/http/gateway.md) instead. It can be used in web apps thanks to [Verified Fetch](https://www.npmjs.com/package/@helia/verified-fetch)

Reviewer: I think the verified fetch part is a bit misleading. 
```suggestion
If you are just interested in retrieval, see implementation-agnostic [HTTP Gateway](../reference/http/gateway.md) instead.
```
